### DISCUSSION TITLE
Google fonts

### DISCUSSION THREAD

**Discussion by Noicom** [webstudio-is/webstudio-community#101] [Q&A]
Hello, is there / will be a possibility to use google fonts instead of uploading from computer? thx!

**Comment by AndyHeyAdams** [comment]
May I also ask if there's a way to implement material design icons? When I paste the css code provided by google in the html element I always get an error of sorts. I have zero expertise in this field, I'm just a UI designer...

**Reply by kof** [reply]
We recommend getting SVG and copying these into HTML Embed component. It is less convenient but this makes sure your performance doesn't suffer. Font icons are a huge problem for performance.

In the future we will think of how to make this easier.

**Comment by kof** [comment]
@Noicom currently you would have to download the font from google and upload it to webstudio manually. We will make it easier in the future iterations without compromising privacy.
